pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -28,6 +32,7 @@ import (
 const (
 	AccountClaimed          = "AccountClaimed"
 	AccountUnclaimed        = "AccountUnclaimed"
+	OUConfigMapName         = "aws-account-operator-ou-mapping"
```

Comment:
There are a few other things that the operator could use a config map for. Can we make a single generic config map?  

cc @rogbas

Reply:
agreed, I will change this